contracts/storagetrie/StorageTrieStoreTest.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity 0.4.24;
+
+contract StorageTrieStoreTest {
+    struct Identity {
+        string publicKey;
+        string hash;
+    }
+
+    address public owner; // ex : 0xe5e4a8f4ecc2b6298be33fed07a09599db4e46fa
+
+    string public rootCaCertificate; // ROOT_CA
+    mapping(string => string) caCertificates; // CA_C1
+
+    mapping(string => Identity) identites;
+
+    modifier onlyOwner {
+        require(msg.sender == owner);
+        _;
+    }
+
+    constructor()
+    public {

```

Comment:
1. Only long function declarations separate lines. 
2. If separated, keywords like "public" need 4 spaces indentation.
Refer: https://solidity.readthedocs.io/en/v0.5.12/style-guide.html#function-declaration

Reply:
Thanks for the review, updated!